README.md

```diff
@@ -24,6 +75,17 @@
 
 2. A WIP [Token Allowance Agent ](https://github.com/beneviolabs/ft-allowance-agent/blob/main/0.0.1/agent.py)that capitalizes on market volatility to grow your wealth, by determining which tokens to periodically swap into stablecoins to secure gains without reducing your portfolio below some minimum USD value. Realize these gains for yourself, or setup a conditional recurring allowance for your crypto curious friends & family.
 
+### Deleting a trading/proxy account
+
+1. Add your main account public key to the proxy account with full access permissions
+```
+near contract call-function as-transaction <mainaccount>.auth-v0.peerfolio.testnet add_full_access_key json-args '{"public_key": "<main-account-public-key>"}' prepaid-gas '100.0 Tgas' attached-deposit '0 NEAR' sign-as <mainaccount>.testnet network-config testnet sign-with-keychain send
+```
+
+2. Send a delete account transaction signing with your main account key pair
+```
+near account delete-account <mainaccount>.auth-v0.peerfolio.testnet beneficiary <mainaccount>.testnet network-config testnet sign-with-plaintext-private-key --signer-public-key <main-account-public-key> --signer-private-key <main-account-private-key> send
```

Comment:
I would drop this part `--signer-public-key <main-account-public-key> --signer-private-key <main-account-private-key> send`  the public_key is not required, and the CLI will prompt for the private key  and if you want to send it now